### ISSUE TITLE
O-man sub classes

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#1786]
_This task is for the preparation of 2.8 and will not be part of the 2.7 release._

Currently in the glycan classification we have 
* O-linked
  * O-mannose
  * ...

We would like to add the different O-Man cores (M0-M3 and the yeast related structures). For information:
* [Mammalian O-mannosyl glycans: Biochemistry and glycopathology](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6395781/)
* [Recent advancements in understanding mammalian O-mannosylation](https://pubmed.ncbi.nlm.nih.gov/28810660/)
* [O-Mannosylation](https://www.physiol.uzh.ch/en/Glycosylation/Oglycosylations/Omannosylation.html)

It also raises the question if the Hierarchy should be restructured. Currently the O-GalNAc and O-Man cores would be mixed in with the other O-linked types. One strategy could be to have O-GalNAc and O-Man on the second level and move the cores to a third level below these second level nodes. The other alternative could be to have O-GalNAc and O-Man on the top level. The first will have implications both on the classification and GlyGen code. The latter might not be intuitive.

**Comment by edwardsnj** [comment]
These new subtypes were first added in the glygen glycan data for release 2.8, I believe.

See: 

   https://edwardslab.bmcb.georgetown.edu/glycomotifdev/GGC.000107
   https://edwardslab.bmcb.georgetown.edu/glycomotifdev/GGC.000109

and

   https://edwardslab.bmcb.georgetown.edu/glycomotifdev/GGC.000100

They are also present in the release for 2.10 as well.

**Comment by ReneRanzinger** [comment]
@ubhuiyan If that was part of 2.8 why do we not have this in GlyGen search or on the details page (e.g. https://www.glygen.org/glycan/G85856KC) ? Can we make sure that this will be there in 2.10?

**Comment by ubhuiyan** [comment]
@ReneRanzinger If I understand correctly, these types would show up in the "Advanced Search" dropdown and the Type/Subtype information in the General category for the corresponding glycans?

**Comment by ReneRanzinger** [comment]
Yeah, that's the places I was look for them. Maybe it was not processed in 2.8/2.9?

**Comment by ReneRanzinger** [comment]
@ubhuiyan did we update production? I am now seeing "O-Mannose" and "O-Mannose Core" in the advanced search. The core M1 example above (https://www.glygen.org/glycan/G85856KC) still does not have the type, but others do.

Also what is the difference between O-Mannose and O-Mannose Core?

**Comment by edwardsnj** [comment]
I'm looking at the classification definitions for these two subtypes - I think I originally intended "O-Mannose" to be a single monosaccharide (but was later told it could be longer chains - yeast I think), and "O-Mannose core" was for the O-mannose structures shown in the CCRC pathways at "[O-glycans (non-GalNAc core)](https://glycoenzymes.ccrc.uga.edu/Glycomics3/)". 

See the pages here: [Subtype: O-mannose core](https://edwardslab.bmcb.georgetown.edu/glycomotifdev/GGC.000107) and [Subtype: O-mannose](https://edwardslab.bmcb.georgetown.edu/glycomotifdev/GGC.000109).

I think we could comfortably use the same type "O-linked", subtype "O-mannose" for both of these groups of glycans. 

The structure G85856KC requires an additional beta-4 Gal to meet the definition of "O-mannose core". The non-mannose residue blocks the application of the O-mannose subtype.

**Comment by ReneRanzinger** [comment]
We can discuss the merging of the two next Wednesday. 

Regarding G85856KC. If you look at this paper (https://pmc.ncbi.nlm.nih.gov/articles/PMC6395781/) figure 1: the M1 core is just a disaccharine.

**Comment by ubhuiyan** [comment]
@ReneRanzinger I don't see them on my end. I was planning to bring this up in the GW internal meeting on Monday to ask Robel how he ingests that data. 

<img width="1368" height="490" alt="Image" src="https://github.com/user-attachments/assets/48b67ea8-6dd3-4959-9c09-cbbdc7f1a214" />

**Comment by ReneRanzinger** [comment]
@ubhuiyan  Select O-linked and they will show up in the Glycan Sub Type

**Comment by ReneRanzinger** [comment]
There is also #2801 and #2062.

**Comment by ubhuiyan** [comment]
**Update (1/21/26):** 

Nathan will receive the descriptions for all the O-man core (0-3) subtypes and yeast mannan type and implement for 2.12.

Urnisha will make sure the update is reflected during 2.12 beta/prd testing.
